src/controllers/cohort.js

```diff
@@ -1,15 +1,13 @@
-import { createCohort, getCohorts } from '../domain/cohort.js'
+import { createCohort, getCohorts, getCohort } from '../domain/cohort.js'
 import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 
 export const create = async (req, res) => {
-  const { cohort_name: cohortName } = req.body
```

Comment:
These deletions in the create func look to me incorrect - check in with your team about them

Reply:
readded the code, not sure how it was deleted